### PR TITLE
Added section about multiple questions in DoH

### PR DESCRIPTION
**Pull request by marciocloudflare**
Hi @vavrusa. This is the PR I made regarding how to ask multiple questions in DoH. Could you check it and see if everything makes sense? Thanks!

### CODE REVIEW

File: products/1.1.1.1/src/content/dns-over-https/request-structure.md [hunk 1]
```diff
@@ -23,6 +23,18 @@ When making requests using `POST`, the DNS query is included as the message body
 
 When making requests using `GET`, the DNS query is encoded into the URL. An additional URL parameter of `ct` should indicate the MIME type (see below).
 
+## Send multiple questions in a query
+
+Sending more than one question when making requests using POST depends on the HTTP version used, as each DNS query maps to exactly one HTTP request. HTTP/2 and HTTP/3 have multiplexing and you can start multiple requests concurrently. HTTP/2 is, in fact, the minimum recommended version of HTTP for use with DNS over HTTPS (DoH). This is not specific to 1.1.1.1, but rather how DoH works.
```
Comment (vavrusa) [inline products/1.1.1.1/src/content/dns-over-https/request-structure.md]: delete `using POST`

**Review by vavrusa — CHANGES_REQUESTED** [review]

**Review by vavrusa — APPROVED** [review]

File: products/1.1.1.1/src/content/dns-over-https/request-structure.md [hunk 2]
```diff
@@ -23,6 +23,18 @@ When making requests using `POST`, the DNS query is included as the message body
 
 When making requests using `GET`, the DNS query is encoded into the URL. An additional URL parameter of `ct` should indicate the MIME type (see below).
 
+## Send multiple questions in a query
+
+Sending more than one question when making requests depends on the HTTP version used, as each DNS query maps to exactly one HTTP request. HTTP/2 and HTTP/3 have multiplexing and you can start multiple requests concurrently. HTTP/2 is, in fact, the minimum recommended version of HTTP for use with DNS over HTTPS (DoH). This is not specific to 1.1.1.1, but rather how DoH works. 
+
+You can learn more about how DoH works in the RFC8484 document, more specifically [the HTTP layer requirements](https://datatracker.ietf.org/doc/html/rfc8484#section-5.2).
+
+**Sending multiple questions example:**
```
Comment (pedrosousa) [inline products/1.1.1.1/src/content/dns-over-https/request-structure.md]: I'd use a shorter Heading 3 (`### Example`) or a full sentence (`The following example sends two questions in a query:`), avoiding the bold.

File: products/1.1.1.1/src/content/dns-over-https/request-structure.md [hunk 3]
```diff
@@ -23,6 +23,18 @@ When making requests using `POST`, the DNS query is included as the message body
...
+
+```sh
+$ curl --http2 -H 'accept: application/dns-json' https://1.1.1.1/dns-query?name=cloudflare.com --next --http2 -H 'accept: application/dns-json' https://1.1.1.1/dns-query?name=cloudflare.com
```
Comment (pedrosousa) [inline products/1.1.1.1/src/content/dns-over-https/request-structure.md:35]: Should the two questions be about the same domain (`cloudflare.com`)? Looks odd.

File: products/1.1.1.1/src/content/dns-over-https/request-structure.md [hunk 4]
```diff
@@ -23,6 +23,18 @@ When making requests using `POST`, the DNS query is included as the message body
 
 When making requests using `GET`, the DNS query is encoded into the URL. An additional URL parameter of `ct` should indicate the MIME type (see below).
 
+## Send multiple questions in a query
+
+Sending more than one question when making requests depends on the HTTP version used, as each DNS query maps to exactly one HTTP request. HTTP/2 and HTTP/3 have multiplexing and you can start multiple requests concurrently. HTTP/2 is, in fact, the minimum recommended version of HTTP for use with DNS over HTTPS (DoH). This is not specific to 1.1.1.1, but rather how DoH works. 
+
+You can learn more about how DoH works in the RFC8484 document, more specifically [the HTTP layer requirements](https://datatracker.ietf.org/doc/html/rfc8484#section-5.2).
```
Comment (pedrosousa) [inline products/1.1.1.1/src/content/dns-over-https/request-structure.md]:
```suggestion
You can learn more about how DoH works in RFC 8484, more specifically [the HTTP layer requirements](https://datatracker.ietf.org/doc/html/rfc8484#section-5.2).
```

**Review by pedrosousa — APPROVED** [review]

File: products/1.1.1.1/src/content/dns-over-https/request-structure.md (hunk 2 above)
Comment (marciocloudflare) [reply products/1.1.1.1/src/content/dns-over-https/request-structure.md]: Will do

File: products/1.1.1.1/src/content/dns-over-https/request-structure.md (hunk 3 above)
Comment (marciocloudflare) [reply products/1.1.1.1/src/content/dns-over-https/request-structure.md:35]: I'll double check with the engineer about this

**Pull request merged**